### PR TITLE
fix(packages): add @unlock-protocol/hardhat-plugin to packages build

### PR DESCRIPTION
**Pull request by clemsos**
# Description

Add missing ref to hardhat-plugin in clean/build workflow for packages 

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
btw is that normal that we are checking the `dist`folder in unlock-assets ?
 https://github.com/unlock-protocol/unlock/tree/master/packages/unlock-assets/dist

**Comment by searchableguy** [comment]
@clemsos It's been there for a while but we should remove these generated files (same for build folder in the same package).

File: packages/contracts/package.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
     "publish:npm": "yarn prepublish && npm publish"
   },
   "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^3.1.0",
     "@primitivefi/hardhat-dodoc": "0.2.3",
     "@unlock-protocol/hardhat-helpers": "workspace:^",
     "@unlock-protocol/tsconfig": "workspace:./packages/tsconfig",
     "@unlock-protocol/types": "workspace:^",
-    "@nomiclabs/hardhat-etherscan": "^3.1.0",
```
Comment (searchableguy) [inline packages/contracts/package.json:26]: Been wondering why this dependency keep changing place in the hierarchy 🤔

**Review by searchableguy — APPROVED** [review]

File: packages/contracts/package.json (hunk 1 above)
Comment (clemsos) [reply packages/contracts/package.json:26]: yarn makes deps follow alphabetical order - I added it manually. Which makes me think I need to remove that caret in the version.

**Pull request merged**